common/lib/capa/capa/responsetypes.py

```diff
@@ -1240,12 +1240,14 @@ def mc_setup_response(self):
                     i += 1
                 # If using the masked name, e.g. mask_0, save the regular name
                 # to support unmasking later (for the logs).
-                if self.has_mask():
-                    mask_name = "mask_" + str(mask_ids.pop())
-                    self._mask_dict[mask_name] = name
-                    choice.set("name", mask_name)
-                else:
-                    choice.set("name", name)
+                # Masking is currently disabled so this code is temporarily commented, as
```

Comment:
So this is code that @nparlante wrote. I don't know how this code actually managed to make it to merging -- variables here are not used anywhere else. The comment about ti being disabled is copied from an existing, similar comment elsewhere in the code.

I'd be happy to refactor to just remove all references to masking in total. I don't know if Nick ever plans to finish the implementation.